src/main/java/uk/gov/hmcts/ccd/CoreCaseDataApplication.java

```diff
@@ -33,4 +37,9 @@ public static void main(String[] args) {
         SpringApplication.run(CoreCaseDataApplication.class, args);
     }
 
+    @Bean
+    public Clock uTCClock() {
+        return Clock.system(ZoneOffset.UTC);
```

Comment:
could we replace with: 
`Clock.systemUTC()`?

Reply:
can we notify QAs we've made some changes on the way we retrieve current time ? Better doing some regression testing on this

Reply:
sure but no changes tbh. 
LocalDateTime.now(ZoneOffset.UTC) => LocalDateTime.now(Clock.systemUTC())

Reply:
ok then if you are 100% confident ignore my request